### PR TITLE
Force / Enforce security if service is not available

### PR DESCRIPTION
**Pull request by pvizeli**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request: https://github.com/home-assistant/developers.home-assistant/pull/860
- Link to cli pull request: https://github.com/home-assistant/cli/pull/285

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast supervisor tests`)
- [x] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [ ] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

File: supervisor/api/supervisor.py [hunk 1]
```diff
@@ -115,6 +117,7 @@ async def info(self, request: web.Request) -> Dict[str, Any]:
             ATTR_DEBUG_BLOCK: self.sys_config.debug_block,
             ATTR_DIAGNOSTICS: self.sys_config.diagnostics,
             ATTR_CONTENT_TRUST: self.sys_config.content_trust,
+            ATTR_FORCE_SECURITY: self.sys_config.force_security,
```
Comment (agners) [inline supervisor/api/supervisor.py:120]: `force_security` can be lot of things, how about `mandatory_content_trust`?

Comment (pvizeli) [reply supervisor/api/supervisor.py:120]: It's not all about that. We have other places which we need make decisions like pwnd flows. It's allowed to use insecure secrets? Ect. With enabling we handle security issues straight forward and that can have negative side effects if you are not aware

Comment (pvizeli) [reply supervisor/api/supervisor.py:120]: I add a new commit to show the plan

File: supervisor/resolution/checks/addon_pwned.py [hunk 2]
```diff
@@ -38,11 +37,13 @@ async def run_check(self) -> None:
             # check passwords
             for secret in secrets:
                 try:
-                    if not await check_pwned_password(self.sys_websession, secret):
-                        continue
+                    await check_pwned_password(self.sys_websession, secret)
+                    continue
```
Comment (frenck) [inline supervisor/resolution/checks/addon_pwned.py:41]: This now break the logic below

Comment (ludeeus) [reply supervisor/resolution/checks/addon_pwned.py:41]: We only want that part if `check_pwned_password` raises `PwnedSecret` and then the logic is applied?

Comment (ludeeus) [reply supervisor/resolution/checks/addon_pwned.py:41]: That part of the logic should probably be moved to the `PwnedSecret` exception to make it clearer

Comment (pvizeli) [reply supervisor/resolution/checks/addon_pwned.py:41]: I can't follow

Comment (ludeeus) [reply supervisor/resolution/checks/addon_pwned.py:41]:
```
                # Check possible suggestion
                if addon.state == AddonState.STARTED:
                    suggestions = [SuggestionType.EXECUTE_STOP]
                else:
                    suggestions = None
                self.sys_resolution.create_issue(
                    IssueType.PWNED,
                    ContextType.ADDON,
                    reference=addon.slug,
                    suggestions=suggestions,
                )
                break
```
That part from line 50

**Review by ludeeus — APPROVED** [review]

**Pull request merged**